### PR TITLE
Change velero entrypoint

### PR DESCRIPTION
**Pull request by jenting**
For both support install Velero server by `helm install` and `velero install`, change the velero entrypoint to `/velero` as upstream is. Ref https://github.com/vmware-tanzu/velero/blob/master/Dockerfile-velero#L33.

For `helm install`, it supports to change the entrypoint https://github.com/vmware-tanzu/helm-charts/blob/42f79f4bd7846bed7f4d10395a39ada070a3c33b/charts/velero/templates/deployment.yaml#L51.

For `velero install`, it uses hardcode entrypoint path
- https://github.com/vmware-tanzu/velero/blob/c832e52905dc1b00b2fa4354e37b483dc13cbbe8/pkg/install/deployment.go#L146
- https://github.com/vmware-tanzu/velero/blob/c832e52905dc1b00b2fa4354e37b483dc13cbbe8/pkg/install/daemonset.go#L93

Therefore, for support both installation way, change the velero entrypoint to `/velero`.

### CODE REVIEW

**Comment by innobead** [comment]
@c3y1huang @cclhsu please help review. Thanks.

File: caasp-velero-image/config.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+# Greeting...
+#--------------------------------------
+echo "Configure image: [$kiwi_iname]..."
+
+#======================================
+# Imitate upstream image:
+# velero/velero
+# same file structure
+#--------------------------------------
+cp /usr/bin/velero /velero
```
Comment (davidcassany) [inline caasp-velero-image/config.sh]: Why not a symlink? this adds about 50Mb of extra data into the image. Go packages are, in many cases, horribly fat.

File: caasp-velero-image/caasp-velero-image.kiwi [hunk 2]
```diff
@@ -17,7 +17,7 @@
         tag="%%PKG_VERSION%%"
         maintainer="SUSE Containers Team &lt;containers@suse.com&gt;"
         user="nobody">
-        <entrypoint execute="velero"/>
+		<entrypoint execute="/velero"/>
```
Comment (davidcassany) [inline caasp-velero-image/caasp-velero-image.kiwi]: I don't see why this is required, isn't this overwritten in the examples linked within the PR? I am not against it, just trying to understand if this is required. If our package installs velero in `/usr/bin` I'd say the expected approach is to use that by default and just add some compat link for non standard locations.

File: caasp-velero-image/config.sh (hunk 1 above)
Comment (jenting) [reply caasp-velero-image/config.sh]: Good idea. Updated!

File: caasp-velero-image/caasp-velero-image.kiwi (hunk 2 above)
Comment (jenting) [reply caasp-velero-image/caasp-velero-image.kiwi]: Agree, revert this one.

File: caasp-velero-image/config.sh [hunk 3]
```diff
@@ -16,6 +16,6 @@ echo "Configure image: [$kiwi_iname]..."
 # velero/velero
 # same file structure
 #--------------------------------------
-cp /usr/bin/velero /velero
+ln -sf /usr/bin/velero /velero
```
Comment (cclhsu) [inline caasp-velero-image/config.sh]: Do we need to do the same for velero-restic-restore-helper?

Comment (jenting) [reply caasp-velero-image/config.sh]: Good question, I cannot find code in velero with hardcode path `/velero-restic-restore-helper`.
However, for sync with upstream, it's good to do the same for `velero-restic-restore-helper`.

Comment (c3y1huang) [reply caasp-velero-image/config.sh]: Not sure if I understand about this one. If the entrypoint remains in `<<entrypoint execute="velero"/>` and velero package already installs in executable path /usr/bin, why need the extra symbolic link?

**Comment by jenting** [comment]
@davidcassany Please help on review this :bowing_man:

**Pull request merged**